libs/common-system/src/main/java/org/codice/ddf/configuration/AbsolutePathResolver.java

```diff
@@ -25,10 +27,17 @@
 
   private final String path;
 
-  public AbsolutePathResolver(String path) {
+  public AbsolutePathResolver(@Nullable String path) {
     this.path = path;
   }
 
+  /**
+   * This method will attempt to convert the already given path into an absolute path: first using
+   * the system property {@code ddf.home} and then using {@code Path}'s {@code toAbsolutePath}'s
+   * method
+   *
+   * @return Absolute path as a String
+   */
   public String getPath() {
```

Comment:
❓ Why wouldn't use just update this method to be `return getPath(System.getProperty("ddf.home"));`

You could keep a little extra baggage from this older implementation - logging specifically that `ddf.home` is not set would be worthwhile, I suppose - but otherwise, why duplicate logic?

Reply:
@Lambeaux and I felt preserving the log statement was more important.

Reply:
You can still do that by adding the baggage as I suggested.